### DISCUSSION TITLE
The email of my github account has been changed

### DISCUSSION THREAD

**Discussion by orpheeh** [community/community#36706] [Other Feature Feedback, Questions, & Ideas] [answered]
Sorry because i dont speak english well,

I have receive two email who said that the email of my other account (singdev) has been changed.

Now i can't login to this account.

How Can i send request to github for recover my account.

If is necessary i can upgrade to premium account

Please help me!

**Comment by radar-blip** [comment, marked as answer]
For account related issues, see here: [Support page](https://support.github.com/contact/account)

**Comment by ossanna16** [comment]
Hey there @orpheeh and thank you for asking a good question! I'm glad you found our community 🙂
